services/movie_session.py

```diff
@@ -0,0 +1,46 @@
+from django.db.models import QuerySet
+
+from django.utils.datetime_safe import datetime
+
+from db.models import MovieSession, CinemaHall, Movie
+
+
+def create_movie_session(movie_show_time: datetime,
+                         movie_id: int, cinema_hall_id: int) -> MovieSession:
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)
+    movie = Movie.objects.get(id=movie_id)
+    return MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie=movie,
+        cinema_hall=cinema_hall
+    )
+
+
+def get_movies_sessions(session_date: str = None) -> None | QuerySet:
```

Comment:
The return type hint `None | QuerySet` is incorrect for the `get_movies_sessions` function. Since the function always returns a QuerySet, the correct type hint should be `QuerySet[MovieSession]`.